test/db/cmd/undoredo

```diff
@@ -217,16 +235,16 @@ s+
 sj
 EOF
 EXPECT=<<EOF
-[{"offset":0,"current":true}]
-[{"offset":0},{"offset":10,"current":true}]
-[{"offset":0},{"offset":10,"current":true},{"offset":20}]
-[{"offset":0,"current":true},{"offset":10},{"offset":20}]
+[{"offset":0,"cursor":0,"current":true}]
```

Comment:
I can remove the `current` field when it is not true... However I preferred to always include it, so users can know what fields to expect. WDYT?

Reply:
no, keep it.